src/views/Player.vue

```diff
@@ -275,12 +296,39 @@ export default {
     },
 
     listenToPlayButtons(){
-      (this.player.playing == true) ? 
-        this.played.push(this.player.currentTime) :
-        this.paused.push(this.player.currentTime)
+      var status = (this.player.playing) ? "played" : "paused"
+
+      this.playback_journey.push(
+        [
+          String(this.getCurrentDateTime()),
+          status,
+          String(this.player.currentTime)
+        ]
+      )
 
       this.uploadJson()
-      console.log(this.played)
+    },
+
+    checkBrowserSupport(){
+        if(this.hasVideoPlayed == -1 && this.player.playing){
+          const timeBefore = Math.round(this.player.currentTime * 100) / 100;
+
+          setTimeout(() => {
+              const timeAfter = Math.round(this.player.currentTime * 100) / 100;
+
+              if (timeAfter == timeBefore) {
+                // browser unsupported -> show error page
+                this.isBrowserSupported = false;
+                this.hasVideoPlayed = 0;
+              }
+              else this.hasVideoPlayed = 1;
+              this.uploadJson()
+
+              var progressBar = document.querySelectorAll(".plyr__progress")[0]
+              progressBar.firstChild.removeAttribute("disabled");
+
+            }, browser_check_time * 1000);
+        }
```

Comment:
`hasVideoPlayed` can have three values - `-1` `0` `1`
`-1` means don't know
`0` means video didn't play
`1` means video has played

Measuring time at `t` and then at `t+5` seconds. If it's same, video is not seeking forward

Whatever the result, we set `hasVideoPlayed` to either `1` or `0` and don't go into this logic again in future.

Also, rounding the `timeBefore` and `timeAfter` to 2 decimal places for consistency. 

Also, while this logic runs, the progress bar is disabled, so this logic cannot be bypassed.
Once this is complete, we set back the progress bar to be enabled.